package.json

```diff
@@ -200,15 +179,19 @@
         "lines": 98
       }
     },
-    "moduleDirectories": [
-      "node_modules",
-      "app"
-    ],
+    "moduleDirectories": ["node_modules", "app"],
     "moduleNameMapper": {
-      ".*\\.(css|less|styl|scss|sass)$": "<rootDir>/internals/mocks/cssModule.js",
-      ".*\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/internals/mocks/image.js"
+      ".*\\.(css|less|styl|scss|sass)$":
+        "<rootDir>/internals/mocks/cssModule.js",
+      ".*\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$":
+        "<rootDir>/internals/mocks/image.js"
     },
-    "setupTestFrameworkScriptFile": "<rootDir>/internals/testing/test-bundler.js",
+    "setupFiles": [
+      "<rootDir>/internals/testing/shim.js",
```

Comment:
on appveyor react is still throwing a bunch of warnings, doesn't seem like the shim is working